package.json

```diff
@@ -8,8 +8,22 @@
     "start": "next start"
   },
   "dependencies": {
```

Comment:
A lesson to everyone here, this is the *perfect* way to organize your package.json. Nice job :)

Reply:
Vercel gets all the credit here. It made the package.json and I just followed its lead.